docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/ONVIF-protocol.md

```diff
@@ -14,9 +15,28 @@ This Onvif Camera Device Service is developed to control/communicate ONVIF-compl
 The latest version {{version}} of the device service API specifications can be found
 [here](../swagger.md).
 
-
-
-## Onvif Features
+## ONVIF Protocol Properties
+
+| Property | Description | EdgeX Value Type |               
+| -- | -- | -- |  
+| Address | The IP address of the camera. | String |  
+| Port | The internet port of the camera. | String |  
+| EndpointRefAddress | A GUID used to indentify the camera endpoint. | String |  
+| DeviceStatus | The discovery/authorization status of the camera.  | String |  
+| &nbsp;&nbsp;&nbsp;&nbsp; *UpWithAuth* | The device can operate with full authorization. |  |
+| &nbsp;&nbsp;&nbsp;&nbsp; *UpWithoutAuth* | The device only execute commands that do not require authorization. |  |
+| &nbsp;&nbsp;&nbsp;&nbsp; *Reachable* | The device has been discovered, but can not execute anny commands.|  |
+| &nbsp;&nbsp;&nbsp;&nbsp; *Unreachable* | The device can not be found on the network. |  |
+| LastSeen | The time that the device was last seen on the network. | String |  
+| MACAddress | The MAC address of the camera. | String |  
+| FriendlyName | The user defined name for the camera. | String |  
+| Manufacturer | The manufacturer of the camera. | String |
+| Model | The model desciption of the camera. | String | 
+| FirmwareVersion | The firmware version of the camera. Format will vary by manufacturer and model. | String |
+| SerialNumber | The serial number of the camera. | String |
+| HardwareId | The hardware ID of the camera. | String | 
+
```

Comment:
CustomMetadata is weird because it is it's own protocol, that is entirely user configurable. It's not a part of ProtocolProperties, but if I were to address it separately, I would just be repeating info from the CustomMetadata page

Reply:
It is part of protocol properties as per my understanding. This page is to list all the protocol properties that the device service supports. Yes the user can add any data under it but still it belongs to Protocol properties of device onvif service.